src/Smp.Web/Controllers/AccountsController.cs

```diff
@@ -31,24 +33,25 @@ public AccountsController(IUsersRepository usersRepository, IActionsRepository a
         public async Task<IActionResult> ForgottenPassword(string email)
         {
             var user = await _usersRepository.GetUserByEmail(email);
-            if (user == null) return NotFound(new Error("invalid_email", "Email must be linked to an existing user account."));
-
+            if (user == null) return NotFound(new Error("invalid_email",
```

Comment:
Probably best to just put the return on a new line.

Reply:
Addressed in 9902032.